site/docs/plugins/chat-members.md

```diff
@@ -0,0 +1,83 @@
+# Chat Members plugin (`chat-members`)
+
+Automatically store information about users on a chat and retrieve it easilly
+
+## Introduction
+
+In many situations, it is necessary for a bot to have information about all the users of a given chat. Currently, though, the Telegram Bot API exposes no method that allows that to happen.
+
+This plugin does the next best thing: automatically listening to `chat_member` events and storing the `ChatMember` object. It then allows you to use `ctx.chatMembers.getChatMember()` to obtain information about a specific chat member, taking the previously stored information into account.
+
+## Usage
+
+### Storing chat members
+
+You can use a valid grammY [storage adapter](https://grammy.dev/plugins/session.html#known-storage-adapters) or an
+instance of any class that implements the [`StorageAdapter`](https://deno.land/x/grammy/mod.ts?s=StorageAdapter)
+interface.
+
+```typescript
```

Comment:
Needs to be included as TS/JS/Deno

Reply:
Right. Forgot about that. My bad.
I'll do it later today

Reply:
I’ll do it for you.